examples/dswx_s1_sample_runconfig-v3.0.0-rc.1.0.yaml

```diff
@@ -75,7 +75,7 @@ RunConfig:
         ProductIdentifier: DSWX_S1
 
         # Product version specific to output products
-        ProductVersion: "1.0"
+        ProductVersion: "0.1"
```

Comment:
Is this number dictated to us?  It seems like an odd starting point as if we will hundreds of versions.

Reply:
The convention has been to use 0.1 until we actually enter production, then we change to 1.0. After that, any changes to the product version number are supposed to be dictated to us from ADT or project management. The previous 1.0 here was likely a cut and paste error from another example runconfig.